### PR TITLE
Remove perWindow Support

### LINKED ISSUE #60: Remove perWindow Support
`perWindow` hacks around LightweightThemeManager and is bound to break.

### PR DESCRIPTION
**Pull request by derinb**
perWindow hacks around LightweightThemeManager and is bound to break.

Fixes #60

### CODE REVIEW

**Comment by derinb** [comment]
No code addition actually. Just indentation level is changed for some conditional code.

File: extension/modules/personas.js [hunk 1]
```diff
@@ -573,44 +550,24 @@ var PersonaController = {
...
+                PersonaService.changeToRandomPersona(node.getAttribute("category"));
+                break;
+            case "custom":
+                PersonaService.changeToPersona(PersonaService.customPersona);
+                break;
+            default:
+                PersonaService.changeToPersona(JSON.parse(persona));
+                break;
         }
+
```
Comment (wagnerand) [inline extension/modules/personas.js]: Nit: Remove this newline.

File: extension/modules/personas.js [hunk 2]
```diff
@@ -625,32 +582,18 @@ var PersonaController = {
...
+            this._resetTimeoutID = null;
         }
+
+        let t = this;
+        let callback = function() {
+            t._previewPersona(persona)
+        };
+        this._previewTimeoutID =
+            window.setTimeout(callback, this._previewTimeout);
+
```
Comment (wagnerand) [inline extension/modules/personas.js]: This newline too.

File: extension/modules/personas.js [hunk 3]
```diff
@@ -934,16 +871,13 @@ var PersonaController = {
...
-                        item.setAttribute("autocheck", "false");
-                        item.addEventListener("command", PersonaController.useRandomPersonaInListener, true);
-                    }
+                    let item = popupmenu.appendChild(document.createElement("menuitem"));
+                    item.setAttribute("label", this._strings.get("useRandomPersona", [this._strings.get("favorites")]));
+                    // item.setAttribute("type", "checkbox");
+                    item.setAttribute("checked", (PersonaService.selected == "randomFavorite"));
+                    item.setAttribute("autocheck", "false");
+                    item.addEventListener("command", PersonaController.useRandomPersonaInListener, true);
+
```
Comment (wagnerand) [inline extension/modules/personas.js]: This newline too.

**Comment by derinb** [comment]
Updated

**Comment by wagnerand** [comment]
Thanks.

**Pull request merged**